lib/Loader/FileLoader.js

```diff
@@ -239,7 +239,11 @@ class FileLoader {
     } else if (argument.slice(0, 1) === '@') {
       return new Reference(argument.slice(1))
     } else if (argument.slice(0, 1) === '%' && argument.slice(-1) === '%') {
-      return this._container.getParameter(argument.slice(1, -1))
+      if (argument.slice(1, 4) === 'env') {
+        return process.env[argument.slice(5, -2)]
+      } else {
+        return this._container.getParameter(argument.slice(1, -1))
+      }
```

Comment:
May we should create a private method `_getArgumentParameter(argument)` to avoid `if-returns` condensity?

Reply:
Done!